src/zino/scheduler.py

```diff
@@ -40,14 +42,27 @@ def get_scheduler() -> AsyncIOScheduler:
 
 
 @log_time_spent()
-def load_polldevs(polldevs_conf: str) -> Tuple[Set, Set]:
-    """Loads pollfile into process state.
+def load_polldevs(polldevs_conf: str, last_run_time: Optional[datetime] = None) -> Tuple[Set, Set]:
+    """
+    Loads pollfile into process state if it was changed since the last time it
+    was loaded
 
     :returns: A tuple of (new_devices, deleted_devices)
     """
+    if last_run_time:
+        try:
+            st_mtime = pathlib.Path(polldevs_conf).stat().st_mtime
+            changed_time = datetime.fromtimestamp(st_mtime, tz=timezone.utc)
+        except OSError as e:  # noqa
+            _log.error(e)
+            return set(), set()
+
+        if changed_time <= last_run_time:
```

Comment:
If the user replaces the polldevs file with a old copy using eg. the `mv` command, the mtime (modified time) parameter is not changed and will be a timestamp in the past. zino should still consider this as a change and load the older polldevs file.  the solution here is to only skip reading if the new mtime is exactly the same as the previous  mtime

Reply:
i think the correct thing here is to only take mtime into account, and not the timestamp when the config was read. The test will then be to check if mtime is changed and not if its newer than the timestamp when the last read was done.

Reply:
I had not considered that! Very good point, will change that accordingly!